src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -20,27 +20,27 @@ This guide includes:
 <ReadMore>Read more about [using the Cloudflare runtime](/en/guides/integrations-guide/cloudflare/) in your Astro project.</ReadMore>
 ## Prerequisites
 
-To get started, you will need:
-
-- A Cloudflare account. If you don’t already have one, you can create a free Cloudflare account during the process.
-
+To get started, you will need a Cloudflare account. If you don’t already have one, you can create a free Cloudflare account during the process.
```

Comment:
I would keep this the way it was, since that's the pattern we use in all the other guides!